spec/mailers/hypertension_fast_track_pilot_mailer_spec.rb

```diff
@@ -12,19 +12,26 @@
       File.read('spec/support/disability_compensation_form/submissions/with_uploads.json')
     end
 
-    let(:pdf_json_string) do
+    let(:rrd_pdf_json) do
       { 'name' => 'VAMC_Hypertension_Rapid_Decision_Evidence.pdf',
         'confirmationCode' => 'fake_confirmation_code',
-        'attachmentId' => 'L023' }.to_s
+        'attachmentId' => 'L048' }
     end
 
     let(:saved_claim) { FactoryBot.create(:va526ez) }
+    let(:bp_readings_count) { 1234 }
 
     let!(:submission) do
-      create(:form526_submission, :with_uploads, user_uuid: 'fake uuid',
-                                                 auth_headers_json: 'fake auth headers',
-                                                 saved_claim_id: saved_claim.id,
-                                                 form_json: form_json + pdf_json_string)
+      submission = create(:form526_submission, :with_uploads, user_uuid: 'fake uuid',
+                                                              auth_headers_json: 'fake auth headers',
+                                                              saved_claim_id: saved_claim.id,
+                                                              form_json: form_json)
+      form_json = JSON.parse(submission.form_json)
+      form_json['rrd_med_stats'] = { bp_readings_count: bp_readings_count }
+      form_json['form526_uploads'].append(rrd_pdf_json)
+      submission.update!(form_json: JSON.dump(form_json))
+      submission.invalidate_form_hash
```

Comment:
Yea, I thought about calling `add_medical_stats` directly, but that would make this test dependent on the implementation of `disability_compensation_job`. This test setup is for the mailer class, which only requires the contents of `submission .form_json` be set appropriately. An alternative is to modify `form_json` before `create(:form526_submission` is called but that is out of order (since the BP readings are added after the `form_json` is set on the record) and may cause confusion. 
I'll add a comment before line 29 to describe why it's doing this.